adrs/Authors - Books API.md

```diff
@@ -0,0 +1,31 @@
+# How to store book authors in the database?
+
+## Status
+Proposed
+
+## Context
+We have a book and we have an author/authors of this book. How should we store information about author/authors?
+
+## Alternatives
+### One alternative
+As string in the “Book” table - store fullnames as a string array.
```

Comment:
I suggest if it's not too hard to add here mermaid-diagrams with the piece of base you're talking about here, so that the variant is immediately demonstrative.